Cargo.toml

```diff
@@ -18,6 +18,7 @@ categories = ["rendering::graphics-api"]
 # Optional log-rs like macros implementation
 # disabled by default
 log-impl = []
+opengl3 = []
```

Comment:
this is not particullary the way miniquad handle different GL versions.
you can check the instanced for a reference of a feature that is not available on certain GL versions but is still used in miniquad.

Reply:
Yes, it is normally done through a fallback. _however_ opengl2 doesn't support the texture format that is needed for HDR to work. This way I can remove that variant from the TextureFormat if there is a chance that openGL2 gets loaded instead of 3.

I am happy to change how it works but to me this sounded the most safe.